contracts/game/ext/scripts/indexer.sh

```
@@ -246,28 +245,28 @@ echo -e "${GREEN}- Network: ${BOLD}${BLUE}$NETWORK${NC}"
+    --http.cors_origins "*" \
+    --db-dir $DB_DIR \
+    --config $TORII_CONFIG > >(setup_log_handling) 2>&1 &
 fi
 
 # Store the PID
-echo $! > "$PID_FILE"
+echo $! >"$PID_FILE"
 
 # Wait briefly and check if process is still running
 sleep 2
 if ! kill -0 $(cat "$PID_FILE") 2>/dev/null; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)